app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    storage = {}
+
+    def wrapper(*args, **kwargs) -> int:
```

Comment:
The return type of the `wrapper` function is currently set to `int`. This is too restrictive, as the decorator might be used with functions that return different types. Consider using `Any` from the `typing` module as the return type to make the decorator more flexible.